src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+	"block.create.clutch": "Sprzęgło",
+	"block.create.shaft": "Wał",
+	"block.create.encased_belt": "Obudowana taśma",
+	"block.create.encased_shaft": "Obudowany wał",
+	"block.create.encased_fan": "Obudowany wiatrak",
+	"block.create.adjustable_pulley": "Przekładnia cięgnowa",
+	"block.create.nozzle": "Dysza",
+	"block.create.hand_crank": "Korba",
+	"block.create.cuckoo_clock": "Zegar z kukułką",
+	"block.create.creative_motor": "Silnik elektryczny",
+	"block.create.belt": "Taśma mechaniczna",
+	"block.create.millstone": "Koło młyńskie",
+	"block.create.crushing_wheel": "Koło mielnicze",
+	"block.create.drill": "Wiertło mechaniczne",
+	"block.create.portable_storage_interface": "Przenośny interfejs magazynowy",
+	"block.create.harvester": "Podbieracz mechaniczny",
+	"block.create.saw": "Piła mechaniczna",
+	"block.create.water_wheel": "Koło wodne",
+	"block.create.mechanical_press": "Prasa mechaniczna",
+	"block.create.mechanical_mixer": "Mikser mechaniczny",
+	"block.create.deployer": "Ręka mechaniczna",
+	"block.create.basin": "Misa",
+	"block.create.mechanical_crafter": "Rzemielśnik mechaniczny",
+	"block.create.flywheel": "Koło zamachowe",
+	"block.create.furnace_engine": "Silnik spalinowy",
+	"block.create.speed_gauge": "Prędkościomierz",
+	"block.create.stress_gauge": "Stresomierz",
+	"block.create.cart_assembler": "Dźwig wagonowy",
+	"block.create.analog_lever": "Dźwignia analogiczna",
+	"block.create.rotation_speed_controller": "Kontroler prędkości obrotów",
+
+	"block.create.sticky_mechanical_piston": "Lepki tłok mechaniczny",
+	"block.create.mechanical_piston": "Tłok mechaniczny",
+	"block.create.mechanical_piston_head": "Głowa tłoka mechanicznego",
+	"block.create.piston_pole": "Ramie tłoka mechanicznego",
+	"block.create.mechanical_bearing": "Wiertarka mechaniczna",
+	"block.create.clockwork_bearing": "Wiertarka zegarowa",
+	"block.create.rope_pulley": "Krążek linowy",
+	"block.create.rope": "Lina",
+	"block.create.pulley_magnet": "Przekładnia magnesu",
+	"block.create.translation_chassis": "Poziomy korpus",
+	"block.create.rotation_chassis": "Pionowy korpus",
+
+	"block.create.contact": "Wykrywacz pulsu",
+	"block.create.redstone_bridge": "Transmiter pulsu",
+	"block.create.stockswitch": "Wykrywacz pojemności",
+	"block.create.flexcrate": "Skrzynia adaptacyjna",
+	"block.create.extractor": "Ekstraktor",
+	"block.create.belt_funnel": "Rozdzielacz",
+	"block.create.linked_extractor": "Podłączony ekstraktor",
+	"block.create.transposer": "Przekładacz",
+	"block.create.linked_transposer": "Podłączony przekładacz",
+	"block.create.pulse_repeater": "Przekaźnik pulsacyjny",
+	"block.create.flexpulsepeater": "Adaptacyjny przekaźnik pulsacyjny",
+	"block.create.redstone_latch": "Zasilany zatrzask",
+	"block.create.toggle_latch": "Zasilany przełącznik",
+	"block.create.flexpeater": "Adaptacyjny przekaźnik",
+	"block.create.entity_detector": "Taśmowy detektor",
+	"block.create.belt_tunnel": "Tunel szeregowy",
```

Comment:
What about _"Tunel taśmowy"_ instead? This way we keep the _"belt"_ keyword

Reply:
I named it "Tunel szeregowy" cause they work as whole if they are next to each other.

Reply:
But okay, I will rename it I the owner give me permission.